cif_pow.dic

```diff
@@ -6228,6 +6230,63 @@ save_PD_DIFFRACTOGRAM
          through the relevant _pd_diffractogram.* data names. This says that
          the diffractogram was collected under those conditions, with that
          instrument, and that specimen.
+
+         It is not sufficient, for machine-readability, that the _diffrn.id,
+         _pd_instr.id, and _pd_spec.id are given in the same block; they must
+         be explicitly set through _pd_diffractogram.diffrn_id,
+         _pd_diffractogram.instr_id, and _pd_diffractogram.spec_id.
+;
+;
+         data_diffrn_info
+            _audit_dataset.id   09248943-e359-49d2-8d9f-800d6eca3b6d
+
+            _diffrn.id                    'high temperature iron ore'
```

Comment:
As currently defined, the `_diffrn.id` data item cannot contain spaces. Same applies to `_pd_diffractogram.diffrn_id`. Same applies to the two data names in example 1 of this category.

Seeing that other ID data names (e.g. `_pd_instr.id`) do allow spaces, you may want to make things consistent. From my personal experience, allowing whitespaces in arbitrary identifiers is asking for trouble, but I do understand that in some cases this is required for backward compatibility.

Reply:
ooh. No, I agree on the whitespace thing. was just typing without thinking.